Dockerfile

```diff
@@ -1,14 +1,11 @@
 # Needed for DynamoDB Local, which is used for unit tests
-FROM openjdk:15-alpine3.11
+FROM amazoncorretto:20-alpine3.18
```

Comment:
The image we were previously using was deprecated:

<img width="867" alt="image" src="https://github.com/ginger-io/ginger-vault-service/assets/1993853/39c5a13c-da83-41c5-b50e-8908d540045a">